packages/block-library/src/comments/edit/comments-legacy.js

```diff
@@ -57,11 +57,10 @@ export default function CommentsLegacy( {
 			<div { ...blockProps }>
 				<Warning actions={ actions }>
 					{ __(
-						"Comments block: You're currently using this block in legacy mode. " +
-							'The following is just a placeholder, not a real comment. ' +
-							'The final styling may differ because it also depends on the current theme. ' +
-							'For better compatibility with the Block Editor, ' +
-							'please consider switching the block to its editable mode.'
+						"Comments block: You're currently using the legacy version. " +
+							'The following is just a placeholder -- it might look different on the frontend. ' +
```

Comment:
Really nitpicking here: are two consecutive hyphens a conscious stylistic choice? I think maybe those were supposed to be an em dash instead?

Reply:
That's correct -- good spot! 😅